src/components/SwapUkraine.tsx

```diff
@@ -283,33 +283,27 @@ const Swap = () => {
 
   // autoselect from chain based on wallet
   useEffect(() => {
-    LiFi.getChains().then((chains: Chain[]) => {
-      const walletChainIsSupported = chains.some((chain) => chain.id === web3.chainId)
+    if (!fromChainKey && startParamsDefined) {
```

Comment:
This useEffect block is used in multiple places, maybe abstract it into a hook?

Reply:
multiple useEffect hooks in this repo are used *a lot*. I agree it would be nice but this repo has entered maintenance mode. The new transferto repository has been created and the work on the new site using the widget has started.